lms/static/scripts/frontend_apps/components/FilePickerFormFields.js

```diff
@@ -34,9 +35,13 @@ export default function FilePickerFormFields({
   formFields,
   groupSet,
   ltiLaunchURL,
+  extLTIAssignmentId,
 }) {
   /** @type {Record<string,string>} */
   const extraParams = groupSet ? { group_set: groupSet } : {};
+  if (extLTIAssignmentId) {
+    extraParams.ext_lti_assignment_id = extLTIAssignmentId;
+  }
```

Comment:
`extraParams` end up in the URL on `lms/static/scripts/frontend_apps/utils/content-item.js`